auth.go

```diff
@@ -215,18 +218,25 @@ type Central struct {
 	MaxActiveSessions      int32
 	NewSessionExpiresAfter time.Duration
 	Ticker                 *time.Ticker
```

Comment:
I don't think Ticker needs to be public. Also, it needs a better name, like userStoreMergeTicker